pkg/apicapi/apicapi.go

```diff
@@ -423,7 +425,12 @@ func (conn *ApicConnection) runConn(stopCh <-chan struct{}) {
 		}()
 	}
 
-	refreshTicker := time.NewTicker(conn.RefreshInterval)
+	refreshTickTime := conn.RefreshInterval
+	if refreshTickTime == 0 {
```

Comment:
Can we change this to:

if conn.RefreshInterval != 0 {
        refreshTicker := time.NewTicker(conn.RefreshInterval)
        defer refreshTicker.Stop()
}